packages/shared/src/graphql/squads.ts

```diff
@@ -51,10 +51,7 @@ type EditSquadInput = SharedSquadInput & {
   sourceId: string;
 };
 
-type CreateSquadInput = SharedSquadInput & {
-  postId: string;
-  commentary: string;
-};
+type CreateSquadInput = SharedSquadInput;
```

Comment:
Just a question. Is it not possible to create a Squad without a post? Like for when they clicked the Share button and then selected "Create Squad"?

Reply:
Think we remove this in favor of simpler squad creation, but will check with product :)

Reply:
Oh, cool. Thank you for checking with him! 🚢